src/coord/src/catalog.rs

```diff
@@ -124,6 +124,16 @@ pub struct Schema {
     #[serde(skip)]
     pub oid: u32,
     pub items: BTreeMap<String, GlobalId>,
+    pub functions: BTreeMap<String, GlobalId>,
+}
+
+impl Schema {
+    pub fn get_items(&self) -> &BTreeMap<String, GlobalId> {
+        &self.items
+    }
+    pub fn get_functions(&self) -> &BTreeMap<String, GlobalId> {
+        &self.functions
+    }
```

Comment:
Having public accessors for public fields weirds me out a little bit. Whatchya think about just inlining `|schema| &schema.items` etc. below? I think it should just naturally coerce into a fn pointer. Don't worry about it if it takes more than 30s.